__tests__/controllers/users.controller.spec.ts

```diff
@@ -84,18 +85,43 @@ describe('requesting a user', () => {
     const id = user.id;
     const response = await request(app).get(`${API}/users/${id}`);
     expect(response.status).toBe(200);
+    expect(response.body).not.toHaveProperty('password');
   });
 });
 
 describe('creating a user', () => {
   it('returns http code 200 and creates the user', async () => {
     const userFields = await factory(User)().create();
 
+    const userRepo = getRepository<User>(User);
+    userRepo.clear();
+
+    expect(await userRepo.count()).toBe(0);
+
     const response = await request(app).post(`${API}/users`).send(userFields);
     expect(response.status).toBe(200);
 
+    expect(await userRepo.count()).toBe(1);
+  });
+
+  it('returns http code 400 if user email already exists', async () => {
+    const userFields = await factory(User)().create();
+
     const userRepo = getRepository<User>(User);
-    expect(await userRepo.count()).toBeGreaterThan(0);
+    userRepo.clear();
+
+    const validResponse = await request(app)
+      .post(`${API}/users`)
+      .send(userFields);
+    expect(validResponse.status).toBe(200);
```

Comment:
Maybe we should only have the next expectation, but I don't have a solid opinion 👍